src/reference/chat/data.json

```diff
@@ -174,6 +174,23 @@
         }
       }
     },
+    "clearMessages": {
+      "description": "Clears all chat messages in the channel. (If you have the `clear_messages` permission)",
```

Comment:
Might be useful to have an annotation for permissions in the chat server methods